storage/ndb/src/kernel/vm/SimulatedBlock.hpp

```diff
@@ -2331,16 +2298,11 @@ class alignas(NDB_CL) SimulatedBlock :
    */
   Uint32 getFirstLDMThreadInstance()
   {
-    if (unlikely(globalData.ndbMtLqhThreads == 0))
-    {
-      return globalData.ndbMtMainThreads +
-             globalData.ndbMtTcThreads;
-    }
-    else
-    {
-      /* Adjust for proxy instance with + 1 */
-      return globalData.ndbMtMainThreads + 1;
-    }
+    return 1;
```

Comment:
Could it be 0 if lqh, tc and main are all 0?

Reply:
Nope, we always have proxy blocks, even in this case, thus it will always be 1.